terraform/lb.tf

```diff
@@ -0,0 +1,59 @@
+####### ALB #######
+
+##### ALB config #####
+resource "aws_alb" "alb-da-wordpress" {
+  name = "alb-da"
+  security_groups = "${aws_security_group.sg-alb.id}"
+  subnets = ["${aws_subnet.public-wp-a.id}", "${aws_subnet.public-wp-b.id}"]
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+# port listener
+resource "aws_alb_listener" "albListeners-wp" {
+  load_balancer_arn = "${aws_alb.alb-da-wordpress.arn}"
+  port = "443"
+  protocol = "HTTPS"
+  default_action {
+    target_group_arn = "${aws_alb_target_group.target-group-alb}"
+    type = "forward"
+  }
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+# target group
+resource "aws_alb_target_group" "target-group-alb" {
+    name = "alb-group"
+    port = 443
+    protocol = "HTTPS"
+    vpc_id = "${aws_vpc.da-wordpress-vpc}"
+}
+
+
+##### SG #####
```

Comment:
Would you be able to create different files for these resources ?
For sg.tf

If you pull from master,
you'll be able to add this code to the existing sg.tf

Reply:
done! ;)